src/Nowruz/pages/search/useSearch.tsx

```diff
@@ -108,15 +108,15 @@ export const useSearch = () => {
     } else {
       id = item.shortname;
     }
-    navigate(`/profile/${type}/${id}/view`);
+    window.open(`/profile/${type}/${id}/view`);

```

Comment:
why `window.open()`?

Reply:
@satanmourner 
Elham is not available, so feel free to make that change.